valgrind.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# cd to the script directory:
+cd "${0%/*}" || { echo "Couldn't cd to ${0%/*}!"; exit 1; }
```

Comment:
There is absolutely no reason to change directory here. If I'm in the install dir, I should be able to do something like `~/prplmesh/valgrind.sh ./bin/test/foo_test`.

Reply:
Removed cd